src/main/java/com/stefancooper/SpigotUHC/ConfigParser.java

```diff
@@ -99,6 +125,14 @@ public void executeConfigurable(Configurable<?> configurable) {
             case TEAM_ORANGE:
                 createTeam(new UHCTeam("Orange", (String) configurable.value(), ChatColor.GOLD ));
                 break;
+            case PLAYER_HEAD_GOLDEN_APPLE:
+                ItemStack apple = new ItemStack(Material.GOLDEN_APPLE, 1);
+                ItemMeta appleMeta = apple.getItemMeta();
+                apple.setItemMeta(appleMeta);
+                ShapedRecipe recipe = new ShapedRecipe(new NamespacedKey(config.getPlugin(), "PLAYER_HEAD"), apple);

```

Comment:
could you make a new file inside `resources` called `Constants.java` and add a `public static String PLAYER_HEAD = "PLAYER_HEAD";` so that this can be used elsewhere, and if we ever change it then we just change it once